_posts/2020-12-06-converting-your-laptop-to-a-janus-modem-using-unetaudio.md

```diff
@@ -0,0 +1,227 @@
+3. Untar the zip file, open a terminal in the download's root folder and type:
+```
+> bin/unet audio
+Modem web: http://localhost:8080/
+```
+4. This should start up the SDOAM and open a browser with a command shell accessing the modem. If the browser does not automatically open, just enter the modem web URL shown above in your browser. At the command shell, you can try transmitting a message:
+```
+> tell 0, 'hello sea!'
+AGREE
+```
+
+![](../assets/img/unetaudio/unetaudio-webif.png)
+
+You should hear the transmission from your computer speaker! If you don’t, check your speaker volume and try again.
+
+Great, you have just transmitted a message using the default settings of Unet audio.
+
+> NOTE: If you have two computers with speakers and microphones, you can run Unet audio on both, and communicate between the two. If you happen to have only one computer handy,  we can get one Unet audio instance to transmit and receive at the same time as detailed in [section 15.4. Transmitting & receiving using Unet audio](https://unetstack.net/handbook/unet-handbook_physical_service.html) of the [Unet handbook](https://unetstack.net/handbook/unet-handbook_preface.html).
+
+
+## Transmitting & receiving JANUS frames
+
+The physical layer in Unet audio typically supports 3 logical channels . The first two are used for CONTROL and DATA, and the third, by default, is configured as JANUS. The CONTROL channel uses FH-BFSK (frequency-hopping binary frequency shift keying) by default, which is very close to JANUS, but not exactly the same. Let us take a look at the differences between the two and how to make your transmissions fully JANUS compliant.
+
+If you simply type `phy` on the web shell, you get a list of physical layer parameters for Unet audio:
+
+```
+> phy
+« Physical layer »
+
+Provides software-defined physical layer communication services (including error detection & correction).
+
+[org.arl.unet.DatagramParam]
+  MTU ⤇ 31
+  RTU ⤇ 31
+
+[org.arl.unet.phy.PhysicalParam]
+  maxPowerLevel ⤇ 0.0
+  minPowerLevel ⤇ -138.0
+  propagationSpeed = 1500.0
+  rxEnable = true
+
+[org.arl.yoda.ModemParam]
+  adcrate ⤇ 48000.0
+  dacrate ⤇ 96000.0
+  fullduplex = false
+  loopback = false
+  model ⤇ Unet audio
+  vendor ⤇ UnetStack
+```
+
+> NOTE: For brevity, we have omitted the baseband service, scheduler service parameters and some of the non-relevant parameters in the listings given in this article. Even then, there are many parameters that allow you to configure the SDOAM. We cannot cover each parameter in detail here, but we encourage you to explore the help pages for the parameters by simply typing `help phy` followed by the parameter name.
+
+Unet audio defines three types of frames:
+
+- CONTROL frame = 1
+- DATA frame = 2
+- JANUS frame = 3
+
+To get the details of each type of frame, you can use indexed parameters. For getting the details of CONTROL channel, type `phy[1]`.
```

Comment:
In the handbook, we always use `phy[CONTROL]` instead of `phy[1]`. Shall we keep to that?

Reply:
I felt the same when watching the video. CONTROL and JANUS would be clearer.

Reply:
That was what I wrote first. But then I tried it, and it only works with `phy[CONTROL]` but not for `phy[JANUS]`. I assumed we don't do JANUS as it is a spec and not really "CONTROL" or "DATA" (which are more generic). So, it was an active decision to not use it just for CONTROL and `3` for JANUS as that might be confusing.

Reply:
Added that along with the current text for CONTROL instead of replacing it.